app/src/main/java/app/foodfinderapp/MainActivity.kt

```diff
@@ -27,20 +25,28 @@ import com.google.firebase.auth.FirebaseAuth
 import com.google.firebase.firestore.ktx.firestore
 import com.google.firebase.ktx.Firebase
 import androidx.recyclerview.widget.RecyclerView
+import app.foodfinderapp.RestaurantService
 
 
-
-
+/**
```

Comment:
Added JavaDoc comments to MainActivity class to improve readability and maintainability.

[Source](https://kotlinlang.org/docs/coding-conventions.html#documentation-comments)